modules/consensus/setdb.go

```diff
@@ -180,6 +182,9 @@ func (db *setDB) rmItem(bucket string, key interface{}) error {
 	if build.DEBUG && !db.checkConsistencyGuard() && build.Release != "testing" {
 		panic(errNotGuarded)
 	}
+	if build.DEBUG && key == nil {
+		panic(errNilInterface)
+	}
```

Comment:
I don't think that this check is necessary. Won't it panic anyway if you pass in a nil interface?